src/build/buildExecutor.ts

```diff
@@ -66,6 +80,42 @@ export class BuildExecutor {
         return buildResult;
     }
 
+    public startLanguageServer(input: BuildInput, buildUserToken: string) {
+        let buildParameters = this.getBuildParameters(undefined, input, buildUserToken);
+        if (this._environmentController.userType === UserType.MicrosoftEmployee) {
+            buildParameters.envs['DOCS_OPS_TOKEN'] = buildUserToken;
+        }
+        let command = this._binary;
+        let args = buildParameters.serveCommand.split(" ");
+        let options = { env: buildParameters.envs, cwd: this._cwd };
+        let optionsWithFullEnvironment = {
+            ...options,
+            env: {
+                ...process.env,
+                ...options.env
+            }
+        };
+        let serverOptions: ServerOptions = {
+            run: {
+                command,
+                args,
+                options: optionsWithFullEnvironment
+            },
+            debug: {
+                command,
+                args,
+                options: optionsWithFullEnvironment
+            },
+        };
+
+        let clientOptions: LanguageClientOptions = {};
+
+        this._eventStream.post(new BuildProgress(`Starting language server using command: ${command} ${args.join(' ')}`));

```

Comment:
minor： use the `buildParameters.serveCommand` directly to avoid unnessesary join